### PR TITLE
Add small tutorial about installation on ArchLinux

### PR DESCRIPTION
**Pull request by petermax2**
I just added a small tutorial in `/doc/tutorials`.

### CODE REVIEW

**Comment by markus2330** [comment]
Can one of the admins verify if this patch should be build?

File: doc/tutorials/installation-archlinux.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Installation on ArchLinux #
+
+* Create a build directory and compile the library
+
+
+	mkdir build
+
+	cd build
+
+	cmake ..
```
Comment (markus2330) [inline doc/tutorials/installation-archlinux.md]: sounds very similar to generic instructions, maybe linking to them is better.

File: doc/tutorials/installation-archlinux.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+	kdb: error while loading shared libraries: libelektratools.so: cannot open shared object file: No such file or directory
+
+You need to create a configuration file under `/etc/ld.so.conf.d/` (e.g. `/etc/ld.so.conf.d/elektra.conf`).
+
+Add the path where the library has been installed
+
+	/usr/lib/local/
+
+and run `ldconfig` as root.
```
Comment (markus2330) [inline doc/tutorials/installation-archlinux.md]: That information is useful, but also generic (not arch specific), maybe doc/INSTALL.md is more appropriate.

**Comment by markus2330** [comment]
start build

**Comment by markus2330** [comment]
Did you look at the [arch linux packages](https://aur.archlinux.org/packages/elektra/)? They seem to be up to date.

**Comment by markus2330** [comment]
add to whitelist

**Comment by petermax2** [comment]
I checked the PKGBUILD in the AUR. It refers to release 0.8.9, but not the git mainstream. Maybe I am going to upload a elektra-git package to the AUR tomorrow.
I will move the content to `doc/INSTALL.md`

File: doc/tutorials/installation-archlinux.md (hunk 2 above)
Comment (petermax2) [inline doc/tutorials/installation-archlinux.md]: Should I move the content to `doc/INSTALL.md`?

**Comment by markus2330** [comment]
Having a elektra-git package sounds useful.

File: doc/tutorials/installation-archlinux.md (hunk 2 above)
Comment (markus2330) [inline doc/tutorials/installation-archlinux.md]: Yes, please.

**Comment by petermax2** [comment]
Someone has already uploaded a git package to the AUR

https://aur.archlinux.org/packages/elektra-git/﻿

Am 14. Februar 2015 um 14:12 schrieb markus2330 <notifications@github.com>:

**Comment by markus2330** [comment]
Hello,

Peter Nirschl wrote:
> Someone has already uploaded a git package to the AUR
> 
> https://aur.archlinux.org/packages/elektra-git/﻿

Nice.

Btw. there are two irrelevant dependencies: docbook-xsl (only needed in 0.7) 
and tcl (there is only a plugin named like this, but the tcl interpreter is 
never needed, even when the plugin is used).

best regards

**Pull request closed without merging**